src/index.js

```diff
@@ -92,7 +92,10 @@ export default function PrerenderLoader (content) {
 async function prerender (parentCompilation, request, options, inject, loader) {
   const parentCompiler = getRootCompiler(parentCompilation.compiler);
   const context = parentCompiler.options.context || process.cwd();
-  const entry = './' + ((options.entry && [].concat(options.entry).pop().trim()) || path.relative(context, parentCompiler.options.entry));
+  const entry = './' + (
+    (options.entry && [].concat(options.entry).pop().trim()) ||
```

Comment:
Can probably safely use `options.entry` here since it's a user-supplied option rather than a config value - if someone passes `entry[]=1&entry[]=2` it should really be erroring out like it does today rather than silently ignoring the first entry haha.

Reply:
I am a bit concerned about the assumptions made for an array and object entry. From [webpack docs](https://webpack.js.org/concepts/entry-points/), having an array `entry` would make all elements as the source for a single chunk, so all the entry points should be used for the compile. For an object, while the suggestion is `main` as the main entry, the config is such that those keys would be the names of the resulting bundles, and there is no guarantee that the first (alphabetical) key is the entry point. Moreover, if this loader is used with multiple HtmlWebpackPlugins, a SPA for static routes for instance, then maybe the entry point should mirror the webpack chunks that go into that static page.

Reply:
@rkichenama that makes sense, please, check the last commit. Would this idea be better than initial one?